src/snowflake/snowpark/session.py

```diff
@@ -1695,6 +1695,18 @@ def read(self) -> "DataFrameReader":
         supported sources (e.g. a file in a stage) as a DataFrame."""
         return DataFrameReader(self)
 
+    @property
+    def session_id(self) -> int:
+        """Returns an integer that represents the session id of this session."""
+        return self._session_id
+
+    @property
+    def connection(self) -> "SnowflakeConnection":
+        """Returns a :class:`SnowflakeConnection` object that allow you to access the connection between current session
```

Comment:
allow -> allows
between current session -> between the current session

Add a period at the end of the sentence.